packages/@uppy/thumbnail-generator/src/index.js

```diff
@@ -365,19 +366,27 @@ module.exports = class ThumbnailGenerator extends Plugin {
   }
 
   install () {
-    this.uppy.on('file-added', this.onFileAdded)
     this.uppy.on('file-removed', this.onFileRemoved)
-    this.uppy.on('restored', this.onRestored)
+    if (this.opts.lazy) {
+      this.uppy.on('thumbnail:request', this.onFileAdded)
```

Comment:
When scrolling a list of non-image files, this results in queue events like this:

<img width="1227" alt="Screen Shot 2019-11-27 at 19 14 59" src="https://user-images.githubusercontent.com/1199054/69740292-399c0600-114a-11ea-9d52-e57bc3696833.png">

Should we add `isPreviewSupported(file.type)` to `onFileAdded`?